InOut/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ if(APPLE)
     check_deps(USE_COREMIDI CoreMIDI_FOUND CoreFoundation_FOUND)
     if(USE_COREMIDI)
         make_plugin(cmidi cmidi.c)
-        target_link_libraries(cmidi PRIVATE CoreMIDI::coremidi CoreFoundation::corefoundation)
+        target_link_libraries(cmidi PRIVATE CoreMIDI::coremidi CoreFoundation::corefoundation libcsound_internals)
```

Comment:
Why not just add that into the `make_plugin` part seeing as every plugin will need the headers? This target is just the headers right? No binary should be linked into the plugins.

Reply:
Most plugins don't need the internal headers though

Reply:
Ok well they need the csound API. so there does need to be some header included. What is in `libcsound_internals`?

Reply:
Then that's the inverse of what should happen no? The plugin should only use the header for the public facing API for Csound. It gets loaded dynamically at runtime. Why would the cmidi plugin need internal headers?

Reply:
cmidi uses midiops.h which isn't part of the public include dirs. I don't think that's the best design but

Reply:
Alright. Maybe there is a kind of internal plugin